std_msgs/Header header

object_recognition_msgs/Table table

curious_table_explorer/RegisteredObject[] objects
